### PR TITLE
7.0.x Change TreeVisitor to allow multiple annotations to be added to a resource

### PR DESCRIPTION
**Pull request by arukshani**

### CODE REVIEW

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/Main.java [hunk 1]
```diff
@@ -42,8 +42,10 @@ public static void main(String... args) throws IOException {
         Root muleRootObj = xmlParser.getRootObj();
         if (xmlParser.getUnIdentifiedElements() != null && !xmlParser.getUnIdentifiedElements().isEmpty()) {
             logger.info("Following Elements are not supported by the converter yet!");
+            logger.info("-----------------------------------------------------------");
```
Comment (pamod) [inline tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/Main.java]: Any reason for adding this info log ?

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java [hunk 2]
```diff
@@ -170,9 +171,18 @@ public void visit(HttpListener listener) {
         if (listener.getAllowedMethods() != null) { //TODO: allowedmethods can come comma separated. Handle that here.
             allowedMethods = listener.getAllowedMethods();
         }
-        /*Create an annotation without attribute values*/
+        /*Create an annotation without attribute values*/ //TODO:Handle multiple annotations
```
Comment (pamod) [inline tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java]: Any reason for having this TODO ?

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/Main.java (hunk 1 above)
Comment (arukshani) [reply tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/Main.java]: Even if the config contains elements that are not supported by the converter, it tries to do the conversion without them. But user should be notified of the elements that are not supported by the converter. Hence info logs.

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java (hunk 2 above)
Comment (arukshani) [reply tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java]: this is fixed in here >> https://github.com/arukshani/product-ei/commit/28dd9ab8606fb847743b8e1a57afadaac79164e1#diff-a7c35609bfca9af1252baf6d499e26feL285

**Pull request merged**